www/src/pages/foundations/spacing.tsx

```diff
@@ -95,6 +95,17 @@ export default function SpacingPage() {
     pixelValue: <PixelCell spacer={value} />,
   }));
 
+  const allUtilityClassesTabel= (prefix) => sizes.map(size => {
```

Comment:
It seems to me that if it is a function, then it should be a verb. As an option: `createUtilityClassesTabel`.
What do you think?